### ISSUE TITLE
compiler segfaults on ubuntu because of global Time::Span variable

### ISSUE THREAD

**Issue by hangyas** [crystal-lang/crystal#2619]
```crystal
$one : Time::Span = 1.hour
def foo
end
```
works well on my arch linux, but the compiler segfaults on ubuntu 16.04 x86_64 and on [https://play.crystal-lang.org/#/r/ywi
](https://play.crystal-lang.org/#/r/ywi)

**Comment by asterite** [comment]
Thanks for reporting, I'll try to fix it in the 0.17 branch.

Workaround:

```crystal
$one : Time::Span
$one = 1.hour
def foo
end
```

**Comment by asterite** [comment]
Fixed in the release/0.17 branch and master
